qa/rpc-tests/test_framework/mininode.py

```diff
@@ -23,7 +23,7 @@
 import time
 import sys
 import random
-from binascii import hexlify, unhexlify
+from util import *
```

Comment:
When you are deciding to rebase the segwit patchset, let me know if you want #7918 merged first (as that would solve this nit).